src/Microsoft.ML.Transforms/Text/WordBagTransform.cs

```diff
@@ -423,7 +423,7 @@ internal sealed class TermLoaderArguments
 
         [Argument(ArgumentType.AtMostOnce, HelpText = "How items should be ordered when vectorized. By default, they will be in the order encountered. " +
             "If by value, items are sorted according to their default comparison, for example, text sorting will be case sensitive (for example, 'A' then 'Z' then 'a').", SortOrder = 5)]
-        public ValueToKeyMappingEstimator.SortOrder Sort = ValueToKeyMappingEstimator.SortOrder.Occurrence;
+        public ValueToKeyMappingEstimator.MappingOrder Sort = ValueToKeyMappingEstimator.MappingOrder.ByOccurrence;
```

Comment:
>Sort [](start = 55, length = 4)

This thing has at least three names: mappingOrder, Sort, order. Could you choose a name for all of them? PS: suggested name: `KeyOrdinality`)

Reply:
but it's all internal here everywhere else I opted for `keyOrdinality`

---
In reply to: [264758563](https://github.com/dotnet/machinelearning/pull/2862#discussion_r264758563) [](ancestors = 264758563)